client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue

```diff
@@ -162,7 +185,22 @@ export default {
         .then(() => this.$emit('deleted'))
     },
 
+    async saveSignLanguage() {
```

Comment:
Why is this async?

Reply:
dont remember yet, but I have removed it now https://github.com/demos-europe/demosplan-core/pull/2212/commits/1d7a229d01a09827e22d361e45e0994d34719979